clang/lib/CodeGen/CGExprScalar.cpp

```diff
@@ -146,6 +146,15 @@ struct BinOpInfo {
       return UnOp->getSubExpr()->getType()->isFixedPointType();
     return false;
   }
+
+  /// Check if the RHS has a signed integer representation.
```

Comment:
Extra '/' in comment line.

Reply:
This is a Doxygen style comment (like the ones for other functions above). So there should be three '/'.

Reply:
oh right!  I was looking at a different file.